src/tests/ftest/util/ServerUtils.py

```diff
@@ -158,24 +158,24 @@ def stopServer(setname=None, hosts=None):
     has spawned.  Doesn't always work though.
     """
 
-    global sessions
+    global SESSIONS
     try:
-        if setname == None:
-            for k, v in sessions.items():
-                v.send_signal(signal.SIGINT)
+        if setname is None:
+            for key, value in SESSIONS.items():
```

Comment:
(pylint-unused-variable) Unused variable 'key'

Reply:
FWIU, you can suppress this pylint error by prefixing `key` with an `_`.